Alicerce/Sources/Logging/Log+StringLogDestination.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Log+StringLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class StringLogDestination: LogDestination {
```

Comment:
Why do we need this `StringLogDestination`. Wouldn't it use a lot of memory if anyone tries to use it?